<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notification</class>
 <widget class="QDialog" name="notification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>561</width>
     <height>451</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#MyLoginForm {
background: red;
}

#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid pink;
border-radius: 8px;
}

QPushButton {
color: white;
  background: gray;

border-width: 2px;
border-color: pink;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 110px;
max-width: 110px;
min-height: 40px;
max-height: 40px;
}
QLabel {
font-weight: bpi,k;
font-size: 13px;
}
QPushButton:hover { background-color: #d49dc8; }
</string>
   </property>
   <property name="title">
    <string>Notification</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Message : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Titre : </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Information</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Attention</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Erreur</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Duree : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>ms</string>
    </property>
    <property name="maximum">
     <number>150000</number>
    </property>
    <property name="value">
     <number>15000</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>124</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher le message</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Type : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>124</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>notification</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>714</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
